NuGet.props

```diff
@@ -17,8 +17,11 @@
     <NoWarn>$(NoWarn);1591</NoWarn>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard1.0' OR '$(TargetFramework)' == 'net45' OR '$(TargetFramework)' == 'net40' OR '$(TargetFramework)' == 'net35'">
+    <DefineConstants>NET_TOO_OLD_VER</DefineConstants>
```

Comment:
Can you update the github actions test run to ensure we test both an older and a newer version? Not sure how hard this is, but would be nice to ensure things functions properly with and without the ifdef being true.